test/fixtures/multipleErrors.json

```diff
@@ -0,0 +1,55 @@
+[
+  {
+    "type": "google.rpc.BadRequest",
+    "value": {
+      "fieldViolations": [
+        {
+          "field": "document.language",
+          "description": "The document language is not valid"
+        }
+      ]
+    }
+  },
+  {
+    "type": "google.rpc.QuotaFailure",
+    "value": {
+      "violations": [
+        {
+          "subject": "test",
+          "description": "test"
+        }
+      ]
+    }
+  },
+  {
+    "type": "google.rpc.PreconditionFailure",
+    "value": {
+      "violations": [
+        {
+          "type": "test",
+          "subject": "test",
+          "description": "test"
+        }
+      ]
+    }
+  },
+  {
+    "type": "google.rpc.ResourceInfo",
+    "value": {
+      "resourceType": "test",
+      "resourceName": "test",
+      "owner": "test",
+      "description": "test"
+    }
+  },
+  {
+    "type": "google.rpc.ErrorInfo",
+    "value": {
+      "reason": "API_KEY_INVALID",
+      "domain": "googleapis.com",
+      "metadata": {
+        "service": "translate.googleapis.com"
+      }
+    }
+  }
+]
```

Comment:
super small nit: please add an empty line :)  (you see this red thing that shows that the file has no empty line - that means when you `cat` it in the command line, it does not look good)